Mathlib/Algebra/Group/AddChar.lean

```diff
@@ -408,6 +425,16 @@ lemma injective_iff {ψ : AddChar A M} : Injective ψ ↔ ∀ ⦃x⦄, ψ x = 1
 
 end fromAddGrouptoDivisionCommMonoid
 
+section fromAddCommGrouptoDivisionCommMonoid
+variable {G M : Type*} [AddCommGroup G] [DivisionCommMonoid M]
```

Comment:
It looks like this section has exactly the same assumptions as the previous

Reply:
Oh, the above section is misnamed